js/modules/k6/experimental/fs/registry.go

```diff
@@ -0,0 +1,60 @@
+package fs
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+	"sync"
+
+	"github.com/spf13/afero"
+)
+
+// registry is a registry of opened files.
+type registry struct {
+	// files holds a safe for concurrent use map of opened files.
+	//
+	// Keys are expected to be strings holding the files' path.
+	// Values are expected to be byte slices holding the files' data.
+	//
+	// That way, we can cache the file's content and avoid opening too many
+	// file descriptor, and re-reading its content every time the file is opened.
+	//
+	// Importantly, this also means that if the
+	// file is modified from outside of k6, the changes will not be reflected in the file's data.
+	// files map[string][]byte
+	files sync.Map
```

Comment:
What do you think about ching the naming of this to `cache` or `opennedFiles` ?

Reply:
I like it, it's indeed more explicit 👍🏻 

I'll rename it to `openedFiles`, as I don't want to rely too much on the caching terminology as we might evolve the implementation in the future 🙇🏻